src/state/machines/turn.ts

```diff
@@ -371,7 +371,11 @@ export const TurnMachine = setup({
         "user.click.player.endTurn": [{ target: "disaster", guard: "getsDisaster" }, { target: "endOfTurn" }],
         "user.click.market.deck.element": {
           actions: { type: "borrowElement", params: ({ event: { name } }) => name },
-          guard: "canBorrow",
+          guard: and([
+            "belowBorrowLimit",
+            // @ts-expect-error xstate types miss something
+            not({ type: "playerHasElement", params: ({ event: { name } }) => name }),
```

Comment:
The higher order guards I created is an example of how to fix the type issue if you want to remove this expect error